catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/works/v2/ApiV2WorksIncludesTest.scala

```diff
@@ -110,16 +99,300 @@ class ApiV2WorksIncludesTest extends ApiV2WorksTestBase {
+                 |   }
+          """.stripMargin
+          )
+        }
+    }
+  }
+
+  it(
+    "includes a list of production events on a list endpoint if we pass ?include=production") {
+    withV2Api {
+      case (apiPrefix, _, indexNameV2, itemType, server: EmbeddedHttpServer) =>
+        val works = createIdentifiedWorks(count = 2).sortBy { _.canonicalId }
+
+
+        val productionEvents1 = List(ProductionEvent(List(Place("London")), List(Unidentifiable(Person("Bumblebee"))), List(Period("1984")), None))
+        val productionEvents2 = List(ProductionEvent(List(Place("Madrid")), List(Unidentifiable(Person("Bumblebee"))), List(Period("1984")), None))
+        val work0 = works(0).copy(production = productionEvents1)
+        val work1 = works(1).copy(production = productionEvents2)
+
+        insertIntoElasticsearch(indexNameV2, itemType, work0, work1)
+
+        eventually {
+          server.httpGet(
+            path = s"/$apiPrefix/works?include=production",
+            andExpect = Status.Ok,
+            withJsonBody =
+              s"""
+                 |{
+                 |  ${resultList(apiPrefix, totalResults = 2)},
+                 |  "results": [
+                 |   {
+                 |     "type": "Work",
+                 |     "id": "${work0.canonicalId}",
+                 |     "title": "${work0.title}",
+                 |     "production": [ ${production(productionEvents1)}]
+                 |   },
+                 |   {
+                 |     "type": "Work",
+                 |     "id": "${work1.canonicalId}",
+                 |     "title": "${work1.title}",
+                 |     "production": [ ${production(productionEvents2)}]
+                 |   }
+                 |  ]
+                 |}
+          """.stripMargin
+          )
+        }
+    }
+  }
+
+  it(
+    "includes a list of production on a single work endpoint if we pass ?include=production") {
+    withV2Api {
+      case (apiPrefix, _, indexNameV2, itemType, server: EmbeddedHttpServer) =>
+
+        val productionEvent = List(ProductionEvent(List(Place("London")), List(Unidentifiable(Person("Bumblebee"))), List(Period("1984")), None))
+        val work = createIdentifiedWork.copy(production = productionEvent)
+
+        insertIntoElasticsearch(indexNameV2, itemType, work)
+
+        eventually {
+          server.httpGet(
+            path = s"/$apiPrefix/works/${work.canonicalId}?include=production",
+            andExpect = Status.Ok,
+            withJsonBody =
+              s"""
+                 |{
+                 |"@context": "https://localhost:8888/$apiPrefix/context.json",
+                 |     "type": "Work",
+                 |     "id": "${work.canonicalId}",
+                 |     "title": "${work.title}",
+                 |     "production": [ ${production(productionEvent)}]
+                 |   }
+          """.stripMargin
+          )
+        }
+    }
+  }
+
```

Comment:
Not sure if it's elsewhere in this file – do we have tests for work + list endpoints that if we populate all these fields, then call up the API without any includes, they don't appear in the responses?